src/cli/bin.ts

```diff
@@ -69,6 +102,20 @@ yargs
         demandOption: true,
         desc: "The path to the JSON file holding the servers information.",
       });
+      args.option("from-version", {
+        alias: "f",
+        type: "string",
+        demandOption: true,
+        choices: fromVersions,
+        desc: "The version which the content is migrated from.",
```

Comment:
We can be even more concise I think:
```suggestion
        desc: "The version to migrate from.",
```